micro-rdk/src/common/entry.rs

```diff
@@ -205,20 +204,36 @@ pub async fn serve_inner<S>(
         restart,
     )));
 
-    #[cfg(feature = "native")]
     let server_builder = {
+        let http2_port = 12346;
         server_builder.with_http2(
             {
                 let certs = app_client.get_certificates().await.unwrap();
-                let tls_server_config = NativeTlsServerConfig::new(
-                    certs.tls_certificate.as_bytes().to_vec(),
-                    certs.tls_private_key.as_bytes().to_vec(),
-                );
-                let address: SocketAddr = "0.0.0.0:12346".parse().unwrap();
-                let tls = Box::new(NativeTls::new_server(tls_server_config));
-                NativeListener::new(address.into(), Some(tls)).unwrap()
+                let address = SocketAddr::new("0.0.0.0".parse().unwrap(), http2_port);
```

Comment:
⬆️ - Presumably, we need to add support for caching the certs we get back from app just like we cache the robot config. Otherwise, there is no way to bring up the servers in the network partition case. I'll file a ticket for this.